### ISSUE TITLE
[Translation] Updated language strings for Additional HTTP Header

### ISSUE THREAD

**Issue by zero-24** [zero-24/plg_system_httpheader#20]
Hi,

with the latest release I updated the language string for the `Additional HTTP Header` based on the Feedback I got by @brianteeman on the main repo, can you please send PRs or the new translation for the attached string so I can include them in the next release.
The main change was from `additional` to `Force` as in the end we force the headers you set here :)

Fell free to ask any questions that come up :)

Here are the new mention en-GB strings
```
PLG_SYSTEM_HTTPHEADER_ADDITIONAL_HEADER="Force HTTP Headers"
PLG_SYSTEM_HTTPHEADER_ADDITIONAL_HEADER_DESC="Using this you can set different values from the default ones and also force headers. The supported headers are: <br><ul><li>Strict-Transport-Security</li><li>Content-Security-Policy</li><li>Content-Security-Policy-Report-Only</li><li>X-Frame-Options</li><li>X-XSS-Protection</li><li>X-Content-Type-Options</li><li>Referrer-Policy</li><li>Expect-CT</li><li>Feature-Policy</li></ul>"
```

cc
for fr-FR @Sandra97 & @YGomiero
for it-IT @jeckodevelopment
for nl-NL by @pe7er

Thanks for all your translations! 👍

**Comment by jeckodevelopment** [comment]
Italian done

**Comment by pe7er** [comment]
@zero-24 
PR for nl-NL done: https://github.com/zero-24/plg_system_httpheader/pull/22

**Comment by YGomiero** [comment]
PR for fr-FR done: #23

**Comment by zero-24** [comment]
Thanks 👍 All merged for 1.0.10 🎉
